apps/teams-test-app/src/components/LocationAPIs.tsx

```diff
@@ -16,13 +16,13 @@ const GetLocation = (): React.ReactElement =>
     title: 'Get Location',
     onClick: {
       validateInput: input => {
-        if (input.allowChooseLocation === undefined) {
+        if (input && input.allowChooseLocation === undefined) {
```

Comment:
Is validating input necessary anymore if we're doing `location.getCurrentLocation()`? I think it might make more sense to validate `locationProps` in `withCallback` in this instance since input should be allowed to be undefined if we go down the `Promise` route

Reply:
Thank you for the quick review, Jay!!